src/benchpress/datasets/__init__.py

```
@@ -6,14 +6,14 @@
 dataset_registry = DatasetRegistry()
 
 # Import specific dataset types
+from .aime24_hf_dataset import Aime24HfDataset
```

Ruff (0.8.2)

Module level import not at top of file

(E402)